Hunting Queries/ZoomLogs/MultipleRegistrationDenies.yaml

```diff
@@ -0,0 +1,36 @@
+
+id: e119c365-9213-45a1-bbd7-8faf6d103d30
+name: User denined multiple registration events sucessfully registering
+description: |
+  'This hunting query identifies users that have attempted to register for multiple webinars or recordings 
+  but have been deined by the organizer but have then sucessfull registered for an event. The number of events
```

Comment:
denied and after organizer "and then have successfully registered for an event". ;)